### ISSUE TITLE
[BUG] build not working in docker

### ISSUE THREAD

**Issue by valdestron** [ojkelly/yarn.build#52]
**Describe the bug**
I expect yarn build works the same in my terminal and docker socket, seems that it is not.

**To Reproduce**
From monorepo, running *docker build -f services/service/Dockerfile . -t service*

```
FROM node:14.15.1-alpine as bundler
WORKDIR usr/src/app

ADD . .

RUN yarn install
RUN yarn workspace @packages/service build
```

error:

```
/usr/src/app/.yarn/plugins/@ojkelly/plugin-build.cjs:5
TypeError: process.stdout.cursorTo is not a function
    at L.waitUntilDone (/usr/src/app/.yarn/plugins/@ojkelly/plugin-build.cjs:5:13205)
    at Immediate.<anonymous> (/usr/src/app/.yarn/plugins/@ojkelly/plugin-build.cjs:5:13009)
    at processImmediate (internal/timers.js:461:21)
```

**Expected behavior**
Successfully build

**Desktop (please complete the following information):**
 - OS: macOS
 - yarn 2.4.0
 - yarn.build latest

related to https://github.com/ojkelly/yarn.build/issues/12

**Comment by ojkelly** [comment]
Ah, this looks like it’s trying to print it’s dynamic output in a situation where it can’t.

We need to check `process.stdout.cursorTo` exists and fallback to CI output when it doesn’t (such as in docker).

Should be able to get this done within a week.

**Comment by ojkelly** [comment]
This might have been fixed by #53, are you able to install the latest version and check if it's resolved.

**Comment by valdestron** [comment]
Fixed. Thank you.
